sections/header.liquid

```diff
@@ -33,7 +33,7 @@
   }
```

Comment:
I am getting this error on `middle center` https://screenshot.click/10-07-2ads6-1mdfs.mp4

Reply:
Good catch! Fixed. Thanks Sofia :D